spec/features/users_can_edit_a_comment_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe "Users can edit a comment" do
+  include HideFromBullet
```

Comment:
It would be great to explain why this had to be added in the commit message or even inline in the instance.